bitnami/contour/templates/envoy/deployment.yaml

```diff
@@ -260,6 +257,9 @@ spec:
             successThreshold: {{ .Values.envoy.readinessProbe.successThreshold }}
             failureThreshold: {{ .Values.envoy.readinessProbe.failureThreshold }}
           {{- end }}
+          {{- if .Values.envoy.customReadinessProbe }}
```

Comment:
Same comment than in the template above:

```diff
diff --git a/bitnami/contour/templates/envoy/deployment.yaml b/bitnami/contour/templates/envoy/deployment.yaml
index f0e9ce465d..b0b349809f 100644
--- a/bitnami/contour/templates/envoy/deployment.yaml
+++ b/bitnami/contour/templates/envoy/deployment.yaml
@@ -246,7 +246,9 @@ spec:
               {{- end }}
               name: metrics
               protocol: TCP
-          {{- if .Values.envoy.readinessProbe.enabled }}
+          {{- if .Values.envoy.customReadinessProbe }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.envoy.customReadinessProbe "context" $) | nindent 12 }}
+          {{- else if .Values.envoy.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
               path: /ready
```

Reply:
addressed in https://github.com/bitnami/charts/pull/24860/commits/a7b55bb9d4aa96e6afc1407a0eb81a5838a75853